bundles/framework/layerlist/view/LayerViewTabs/LayerList/Filter/Filter.test.js

```diff
@@ -1,49 +1,32 @@
 import React from 'react';
-import { act } from 'react-dom/test-utils';
-import { mount } from 'enzyme';
+import { render } from '@testing-library/react';
+import { beforeEach } from '@jest/globals';
+import '@testing-library/jest-dom';
 import { Filter } from './Filter';
 import { testFilters } from './Filter.test.util';
 import { Controller, LocaleProvider } from 'oskari-ui/util';
-import { Option } from 'oskari-ui';
 import { getBundleInstance } from '../../test.util';
-
 // from https://github.com/ant-design/ant-design/blob/4.3.1/components/select/__tests__/index.test.js
```

Comment:
This comment probably refers to the removed toggleOpen() function so it should be removed as well.